cxx-qt-gen/src/gen_cpp.rs

```diff
@@ -393,16 +389,24 @@ fn generate_properties_cpp(
     for property in properties {
         // Build a CppParameter for the name and type of the property
         let parameter = CppParameter {
-            ident: property.ident.cpp_ident.to_string(),
+            ident: naming::property::name_from_ident(&property.ident)
+                .cpp
+                .to_string(),
             type_ident: &property.type_ident.qt_type,
         };
 
         // Collect the C++ idents for the getter, setter, notify of the property
         //
         // TODO: for now we assume that all properties have a getter/setter/notify
-        let ident_getter = property.getter.as_ref().unwrap().cpp_ident.to_string();
-        let ident_setter = property.setter.as_ref().unwrap().cpp_ident.to_string();
-        let ident_changed = property.notify.as_ref().unwrap().cpp_ident.to_string();
+        let ident_getter = naming::property::getter_from_ident(&property.ident)
+            .cpp
+            .to_string();
+        let ident_setter = naming::property::setter_from_ident(&property.ident)
+            .cpp
+            .to_string();
+        let ident_changed = naming::property::notify_from_ident(&property.ident)
+            .cpp
+            .to_string();
```

Comment:
These three calls should be combined into something like this:
``` rust
let prop_ident = QPropertyName::from(&property.ident);
```
Then you can just use prop_ident.getter.cpp, etc.

Reply:
Also include the above call to name_from_ident into this.

Reply:
So the general thing is that eventually they will be ... just porting the old code was easier to use the free functions. As eventually it'll be `QPropertyName::from(&property)` as in when property is `ParsedQProperty` and not an `extract::Property` ...

Could add some temporary conversions from the "old" style for now to make these free functions private but wasn't sure if it was worth it.

Reply:
Changed properties, i'll sort the others out next week ...

Reply:
Thanks, even though its more work now, I think its still valuable to do this right away.
Especially if we're reusing old code at some point, this might come back to bite us otherwise :see_no_evil: